Challenge02/flor.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>

```

Comment:
I notice there are two opening <head> tags in your code. You should remove one of them to have a properly structured HTML document.